jac-uk/qt
# Bugfix/33 Brief UI Issue When Browser Checks Page Loads

## What's included?
Closes #33  

- [x] Remove `vue-browser-detect-plugin` as it doesn't support Vue3.
- [x] Implement `getBrowserDetect` to replace `vue-browser-detect-plugin`.
- [x] Make the page load smoothly. 

## Who should test?
✅ Product owner
✅ Developers

## How to test?

Preview URL: https://jac-qualifying-tests-develop--bugfix-33-browser-checks-lt4kq84b.web.app/browser-test

Go to "Browser Checks" and check if the page loads smoothly.

## Demo

https://github.com/jac-uk/qt/assets/79906532/734755ed-7744-487b-b362-32f344de955f

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
